bindings/go/src/fdb/transaction.go

```diff
@@ -320,6 +320,12 @@ func (t *transaction) getEstimatedRangeSizeBytes(beginKey Key, endKey Key) Futur
 
 // GetEstimatedRangeSizeBytes will get an estimate for the number of bytes
 // stored in the given range.
+// Note: the estimated size is calculated based on the sampling done by FDB server. The sampling
+// algorithm works roughly in this way: the larger the key-value pair is, the more likely it would
```

Comment:
It sounds like `the larger the key-value pair is`  should be `the larger the key range is`

Reply:
No. The sampling is done at key-value pair basis. Check `StorageServer::byteSampleApply*` functions, specifically `StorageServer::byteSampleApplySet()`, for more details. And the reason behind the recommended usage against large key-range is based on the fact that it's more likely to include more large key-value pairs and thus being more accurate.

Actually even for small keys, statistically it should still be pretty accurate since the actual `sampledSize` for a small key-value pair, once included into the byte sample, is amplified proportionally to the probability that it should have been sampled.